Keyed/Dates.xml

```diff
@@ -4,11 +4,12 @@
   <ClockTime>Heure</ClockTime>
   <ClockDate>Date</ClockDate>
   <ClockYear>Année</ClockYear>
-  <DateReadoutTip>Jours depuis votre arrivée : {0}\nIl y a {1} jours par saison.\nSaison actuelle : {2}</DateReadoutTip>
+  <DateReadoutTip>Jours depuis votre arrivée : {0}\nQuadrum actuel : {4}\nSaison locale : {2}\n\nL'année est divisée en 4 quadrums de {3} jours chacun. Les quadrums sont les mêmes partout, alors que les saisons sont différentes selon les endroits.\n\nSaisons locales pour chaque quadrum :\n{5}</DateReadoutTip>
```

Comment:
J'aurais deux questions. 
La première, le fait de rajouter des {...} ne pose-t-il pas de problème ?
La deuxième, "Saisons locales pour chaque quadrum :\n{5}" ne devrait-il pas être remplacé par "Saisons locales pour chaque quadrum : {5}" ?  Je ne pense pas qu'un retour à la ligne soit adéquat à cet emplacement.

Reply:
C'est bien comme cela. Quid de "quadrum" ?

Reply:
Ah ok, je comprends mieux. Par contre je continue de m'interroger sur ce /n.
Simplement quadrant ou quadrant annuel ?

Reply:
Aucune idéee:
https://fr.wiktionary.org/wiki/quadrum
http://www.lexilogos.com/latin/gaffiot.php?q=quadrum
ou alors "trimestre" https://steamcommunity.com/app/294100/discussions/0/2741975115074278344/

Reply:
Faute de mieux, trimestre, mais je n'aime guère non plus.

Reply:
Je ne parviens pas à trouver les Keyed dans mon fichier d'installation.
Pourrais-tu me mettre la phrase anglaise originale ?

Reply:
```
<?xml version="1.0" encoding="utf-8"?>
<LanguageData>
  <ClockTime>Time</ClockTime>
  <ClockDate>Date</ClockDate>
  <ClockYear>Year</ClockYear>
  <DateReadoutTip>Days passed since your arrival: {0}\nCurrent quadrum: {4}\nLocal season: {2}\n\nThe year is divided into 4 quadrums of {3} days each. Quadrums are the same everywhere, while seasons are different in different places.\n\nLocal seasons for each quadrum:\n{5}</DateReadoutTip>

  <FullDate>{0} of {1}, {2}</FullDate>
  <DateReadout>{0} of {1}, {2}</DateReadout>
  <SeasonFullDate>{0} of {1}</SeasonFullDate>
  <WholeYear>Whole year</WholeYear>
</LanguageData>
```